tx/textual/cbor/cbor.go

```diff
@@ -0,0 +1,230 @@
+// Package cbor implements just enough of the CBOR (Concise Binary Object
+// Representation, RFC 8948) to deterministically encode simple data.
+package cbor
```

Comment:
I am inclined to put this package inside the `internal` folder. I don't think any external people should use it

Reply:
Done.